config/locales/en/unwithdraw/unwithdraw.yml

```diff
@@ -1,4 +1,7 @@
 en:
   unwithdraw:
+    no_managing_editor_permission:
+      title: Only a managing editor can unwithdraw a document
```

Comment:
I'm not sure if this bit of text has been through a content designer. Assuming it hasn't I think it'd be more consistent with other text if it said "Only a managing editor can undo withdraw"

Reply:
Yep, none of the text has really so that's probably the next thing to do